Changelog

```diff
@@ -16,10 +16,25 @@ Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
     - FEATURE: Add option to name each qbittorrent instance (Chocobo1)
     - FEATURE: Add button for sending test email (Thomas Piccirello)
     - FEATURE: Allow torrents to override default share limit action (glassez)
+    - FEATURE: Use Start/Stop instead of Resume/Pause (thalieht) 5d1c2496063b41874e85af0145b153e006fc92d3
+    - FEATURE: Add the Popularity metric (Aliaksei Urbanski) f37d0c486c36e9f6c6e62378385e639feada3085
+    - FEATURE: Focus on Download button if torrent link retrieved from the clipboard (glassez) 7310eec74e6694342e9934ecc524df38d0064bb5
+    - FEATURE: Add ability to pause/resume entire BitTorrent session (glassez) 8ef7d3ec9a56e7445948c37a282629ca5ed551ab
+    - FEATURE: Add an option to set BitTorrent session shutdown timeout (glassez) 2b728b3bc0c525905ef4c6086bc45027eb2150fd
+    - FEATURE: Apply "Excluded file names" to folder names as well (glassez) d89f289f82fb6ff4305841c7d4c7899ffe12f896
+    - FEATURE: Allow to use regular expression to filter torrent content (glassez) 9894f654cf9f9e5ece5e83b55b709c70e398dd02
+    - FEATURE: Allow to move content files to Trash instead of deleting them (glassez) 4149609e785f744c00856d8b4f006b7bc8e14387
+    - FEATURE: Add ability to display torrent "privateness" in UI (ManiMatter) 4d490c84e7b08c87b10b9ba082409832f50084b6
+    - FEATURE: Add a flag in `Peers` tab denoting a connection using NAT hole punching (stalkerok) 63689cf76312303c4323dc1ec16e662444b83b76
     - BUGFIX: Display error message when unrecoverable error occurred (glassez)
     - BUGFIX: Update size of selected files when selection is changed (glassez)
     - BUGFIX: Normalize tags by trimming leading/trailing whitespace (glassez)
     - BUGFIX: Correctly handle share limits in torrent options dialog (glassez)
+    - BUGFIX: Adjust tracker tier when adding additional trackers (Chocobo1) 321d7e5b17b6e4cacf713b4f4409364a5e65873e
+    - BUGFIX: Fix inconsistent naming between `Done/Progress` column (luzpaz) d3315f7cc7f31b1578cf72f07b175a80c204dac9
+    - BUGFIX: Sanitize peer client names (Hanabishi) 2c47f09d7a797c1ea886879749b71198e449e701
+    - BUGFIX: Apply share limits when torrent downloading is finished (glassez) 65d143d4c411e045b3cd1cf847dee831290af799
```

Comment:
I think there should be an "immediately" somewhere in here (the point of this commit).